md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessAPI.java

```diff
@@ -515,15 +500,18 @@ RestWrapper zippedexport(HttpServletResponse resp,
                 out.flush();
             restWrapper = new RestWrapper(processExport, RestWrapper.OK);
             }
-        catch (Exception e) {
+        catch (MetadataException e) {
+            LOGGER.error(e);
             restWrapper = new RestWrapper(e.getMessage(), RestWrapper.ERROR);
+        } catch (IOException io){
+            LOGGER.error(io);
+            restWrapper = new RestWrapper(io.getMessage(), RestWrapper.ERROR);
         }
         return restWrapper;
     }
 
     @RequestMapping(value = {"/import", "/import/"}, method = RequestMethod.POST)
-    public
-    @ResponseBody
+    @ResponseBody public
     RestWrapper importData(@ModelAttribute("fileString")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "importData" is 53 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)